frontend/micro-ui/web/micro-ui-internals/packages/css/src/digitv2/typography.scss

```diff
@@ -12,11 +12,11 @@
       font-size: theme(digitv2.fontSize.heading-xl.desktop);
     }
 
-    @media (min-width: theme(digitv2.screens.tablet)) {
+    @media (min-width: theme(digitv2.screens.tablet)) and (max-width: theme(digitv2.screens.desktop)) {
```

Comment:
@jagankumar-egov Kindly check this query for targeting tablet specific

Reply:
just have only max width and test once
 https://github.com/egovernments/Digit-Core/blob/f0b0515fc808959c1c09c09dc6a36b0fd35e4658/frontend/micro-ui/web/micro-ui-internals/packages/css/src/digitv2/pages/employee/workbench.scss#L19